doc/book/reference/migration/to-v1.1.md

```diff
@@ -0,0 +1,560 @@
+  pipeline for you, which is then stored in `config/pipeline.php`. The generated
+  pipeline contains the following additions:
+
+    - The first middleware in the pipeline is `Zend\Stratigility\Middleware\OriginalMessages`,
+      which injects the incoming request, URI, and response as the request
+      attributes `originalRequest`, `originalUri`, and `originalResponse`,
+      respectively. (This can aid URI generation in nested middleware later.)
+
+    - The second middleware in the pipeline is `Zend\Stratigility\Middleware\ErrorHandler`.
+
+    - The last middleware in the pipeline is `Zend\Expressive\Middleware\NotFoundHandler`.
+
+- Reads your `routes` configuration, and generates a programmatic
+  routing table for you, which is then stored in `config/routes.php`.
+
+- Adds a new configuration file, `config/autoload/programmatic-pipeline.global.php`, 
+  which enables the `programmatic_pipelines` and `raise_throwables`
+  configuration flags outlined above. Additionally, it adds dependency
+  configuration for the new error handlers.
+
+- Inserts two lines before the `$app->run()` statement of your
+  `public/index.php`, one each to include `config/pipeline.php` and
+  `config/routes.php`.
+
+Your `middleware_pipeline` and `routes` configuration are not removed at this
+time, to allow you to test and verify your application first; however, due to
+the configuration in `config/autoload/programmatic-pipeline.global.php`, these
+are now ignored.
+
+If you wish to use Whoops in your development environment, you may add the
+following to a local configuration file (e.g., `config/autoload/local.php`):
+
+```php
+use Zend\Expressive\Container\WhoopsErrorResponseGeneratorFactory;
+use Zend\Expressive\Middleware\ErrorResponseGenerator;
+
+return [
+    'dependencies' => [
+        'factories' => [
+            ErrorResponseGenerator::class => WhoopsErrorResponseGeneratorFactory::class,
+        ],
+    ],
+];
+```
+
+Other things you may want to do:
+
+- The `ErrorHandler` entry could potentially be moved inwards a few layers. As
+  an example, the `ServerUrlMiddleware` has no possibility of raising an
+  exception or error, and could be moved outwards; you could do similarly for
+  any middleware that only injects additional response headers.
+
+- Remove any Stratigility-style error middleware (middleware expecting an error
+  as the first argument). If any specialized error handling should occur, add
+  additional middleware into the pipeline that can catch exceptions, and have
+  that middleware re-throw for exceptions it cannot handle.
+
+- Consider providing your own `Zend\Stratigility\NoopFinalHandler`
+  implementation; this will now only be invoked if the queue is exhausted, and
+  could return a generic 404 page, raise an exception, etc.
+
+## http-interop
+
+Stratigility 1.3 provides the ability to work with [http-interop middleware
+0.2.0](https://github.com/http-interop/http-middleware/tree/ff545c87e97bf4d88f0cb7eb3e89f99aaa53d7a9).
```

Comment:
Maybe better here link to tagged version:
https://github.com/http-interop/http-middleware/tree/0.2.0